WTWSMS/common/landed_titles/landed_titles.txt

```diff
@@ -3475,10 +3477,10 @@ e_byzantium = {
 			independent = yes
 			OR = {
 				culture = helleno_aramean
-				culture = greek
+				culture_group = byzantine
 			}
 			is_adult = yes
-			prestige = 5000
+			prestige = 2000
```

Comment:
These titles are meant to be rare. Stop insisting on things being mistakes when they aren't: it just lowers the credibility of your point to be stating incorrect information. Certainly isn't a good approach if you want to convince,